app/models/bucket-model.coffee

```diff
@@ -51,5 +51,21 @@ global.cobudgetApp.factory 'BucketModel', (BaseModel) ->
     isArchived: ->
       !!@archivedAt && !@paidAt
 
-    iscomplete: ->
-      !!@paidAt
+    isIdea: ->
+      @status == 'draft' && !@archivedAt
+
+    isFunding: ->
+      @status == 'live' && !@archivedAt
+
+    isFunded: ->
+      @status == 'funded' && !@paidAt
+
+    isComplete: ->
+      !!@paidAt && @status == 'funded'
+
+    isCancelled: ->
+      !!@archivedAt && !@paidAt && @status != 'funded'
```

Comment:
hmm i'm pretty sure that a cancelled bucket can also have a status of funded..

Reply:
I did an analysis of the old code base and the current base to see how all these state variables could evolve.

Take a look at [the old state model](https://github.com/cobudget/cobudget-api/wiki/Old-bucket-state-model) and [the new state model](https://github.com/cobudget/cobudget-api/wiki/Current-bucket-state-model). I'm not saying these are correct, so happy to told where they're wrong.

Could you find out how a bucket could be cancelled and still have `status == 'funded'`? 

I think we should add it to the documentation first and then change the code.